src/translations/translationHelper.js

```diff
@@ -0,0 +1,49 @@
+import _ from 'lodash';
+
+const isRootLocaleComment =
+  reply => (reply.depth === 1 && reply.body.length === 4 && reply.body.substring(0, 2) === '# ');
+
+export const toObject = (body) => {
+  const object = {};
+  const lines = body.split(/\n/);
+  lines.forEach((line) => {
+    const message = line.split(':');
+    if (message.length === 2) {
+      object[message[0].trim()] = message[1].trim();
+    }
+  });
+  return object;
+};
+
+export const getMessages = (replies) => {
+  const localesMap = {};
+  const messages = {};
+  const sortedReplies = _.sortBy(replies, 'net_rshares');
+
+  Object.keys(sortedReplies).forEach((key) => {
```

Comment:
I guess instead of two `forEach` with O(2n) complexity you can just use one like this:

```javascript
Object.keys(sortedReplies).forEach((key) => {
     const reply = sortedReplies[key];
     if (isRootLocaleComment(reply)) {
       locale = reply.body.replace('#', '').replace(' ', '').toLowerCase();
       if (locale) {
         const object = toObject(reply.body);
         messages[locale] = { ...messages[locale], ...object }; // es7 way
       }
    }
});
```